docs/shared/guides/remix.md

```diff
@@ -0,0 +1,181 @@
+
+✔ What unit test runner should be used? · vitest
+
+CREATE apps/myapp/project.json
+UPDATE package.json
+CREATE apps/myapp/README.md
+CREATE apps/myapp/app/root.tsx
+CREATE apps/myapp/app/routes/index.tsx
+CREATE apps/myapp/public/favicon.ico
+CREATE apps/myapp/remix.config.js
+CREATE apps/myapp/remix.env.d.ts
+CREATE apps/myapp/tsconfig.json
+CREATE apps/myapp/.gitignore
+CREATE apps/myapp/package.json
+UPDATE nx.json
+CREATE tsconfig.base.json
+CREATE .prettierrc
+CREATE .prettierignore
+UPDATE .vscode/extensions.json
+CREATE apps/myapp/vite.config.ts
+CREATE apps/myapp/tsconfig.spec.json
+CREATE apps/myapp/test-setup.ts
+CREATE apps/myapp-e2e/cypress.config.ts
+CREATE apps/myapp-e2e/src/e2e/app.cy.ts
+CREATE apps/myapp-e2e/src/fixtures/example.json
+CREATE apps/myapp-e2e/src/support/commands.ts
+CREATE apps/myapp-e2e/src/support/e2e.ts
+CREATE apps/myapp-e2e/tsconfig.json
+CREATE apps/myapp-e2e/project.json
+CREATE .eslintrc.json
+CREATE .eslintignore
+CREATE apps/myapp-e2e/.eslintrc.json
+```
+
+## Build, Serve and Test your Application
+
+1. To build your application run:
+
+```{% command="nx build myapp" path="~/acme" %}
+> nx run myapp:build
+
+Building Remix app in production mode...
+⚠️ REMIX FUTURE CHANGE: The behaviors of `CatchBoundary` and `ErrorBoundary` are changing in v2. You can prepare for this change at your convenience with the `v2_errorBoundary` future flag. For instructions on making this change see https://remix.run/docs/en/v1.15.0/pages/v2#catchboundary-and-errorboundary
+⚠️ REMIX FUTURE CHANGE: APIs that provide `formMethod` will be changing in v2. All values will be uppercase (GET, POST, etc.) instead of lowercase (get, post, etc.) You can prepare for this change at your convenience with the `v2_normalizeFormMethod` future flag. For instructions on making this change see https://remix.run/docs/en/v1.15.0/pages/v2#formMethod
+⚠️ REMIX FUTURE CHANGE: The route `meta` export signature is changing in v2. You can prepare for this change at your convenience with the `v2_meta` future flag. For instructions on making this change see https://remix.run/docs/en/v1.15.0/pages/v2#meta
+⚠️ REMIX FUTURE CHANGE: The route `headers` export behavior is changing in v2. You can prepare for this change at your convenience with the `v2_headers` future flag. For instructions on making this change see https://remix.run/docs/en/v1.17.0/pages/v2#route-headers
+⚠️ REMIX FUTURE CHANGE: The `serverModuleFormat` config default option will be changing in v2 from `cjs` to `esm`. You can prepare for this change by explicitly specifying `serverModuleFormat: 'cjs'`. For instructions on making this change see https://remix.run/docs/en/v1.16.0/pages/v2#servermoduleformat
+⚠️ REMIX FUTURE CHANGE: The route file convention is changing in v2. You can prepare for this change at your convenience with the `v2_routeConvention` future flag. For instructions on making this change see https://remix.run/docs/en/v1.15.0/pages/v2#file-system-route-convention
```

Comment:
should we include this in the output? maybe a comment saying removed for brevity?

Reply:
Good idea